components/tag/TagsInput.js

```diff
@@ -0,0 +1,79 @@
+import { useRef } from "react";
+import XMarkIcon from "@heroicons/react/20/solid/XMarkIcon";
+import Input from "../form/Input";
+
+export default function TagsInput({ tags, onTagAdd, onTagRemove }) {
+  const inputRef = useRef(null);
+
+  const keyCode = {
+    comma: 188,
+    backspace: 8,
+  };
+
+  const handleKeyUp = (e) => {
+    const inputValue = inputRef.current.value;
+    const { comma } = keyCode;
+    if (e.keyCode !== comma) return;
```

Comment:
We don't use this code styling anywhere, so I think better to use brackets

```suggestion
    if (e.keyCode !== comma) {
       return;
    }
```